Files/orders/routes.py

```diff
@@ -7,26 +7,46 @@
 cors = CORS(orders_blueprint, resources={r"/foo": {"origins": "*"}})
 
 @orders_blueprint.get('/<int:user_id>')
+@jwt_required()
 @cross_origin(origin='*',headers=['Content- Type','Authorization'])
 def GetOrders(user_id):
+    if user_id != get_jwt_identity():
+        response = jsonify({"message": "You are not authorized to get this user's orders"})
+        response.headers.add("Access-Control-Allow-Origin", "*")
+        return response, 401
     orders = AllOrdersByUser(user_id)
     return orders
 
 @orders_blueprint.get('/<int:user_id>/<int:order_id>')
+@jwt_required()
 @cross_origin(origin='*',headers=['Content- Type','Authorization'])
 def GetOrderByID(user_id, order_id):
+    if user_id != get_jwt_identity():
+        response = jsonify({"message": "You are not authorized to get this user's orders"})
+        response.headers.add("Access-Control-Allow-Origin", "*")
```

Comment:
Don't add that header separately. It's taken care of using that decorator. Adding separately also causes conflicts in frontend. Have tried this before and that's why removed it.